source_files/gpu.c

```diff
@@ -163,12 +167,17 @@ void* startBurn(void *index) {
 }
 
 
-void* fillup(void* array) {
+static void* fillup(void* array) {
```

Comment:
This function can never have worked like intended in the first place.
`array` should be taken as `void**`.